### ISSUE TITLE
Ontraport broke their API

### ISSUE THREAD

**Issue by smikkelsen** [jimmynguyc/ontraport_api#1]
Just an FYI,

Ontraport decided it would be smart to change the behavior of one of their endpoints (GET /objects/meta) and not version it. It now by default returns a 403 error when trying to query for all objects. I've been fighting with them much of the weekend about the breaking change they released on Thursday night (March 16th). According to them, you now have to include an object ID in order to not get an error from that endpoint.

Just throwing it out here in case anyone is relying on that endpoint. I noticed that this gem does implement that endpoint.

Also, on a side note, and actually the reason I could not use this gem in the first place is the "add_sequences" method doesn't actually add sequences, it replaces them with your new list that you pass it. This in my mind is not desired behavior. It should work just like the add_tags.

Maybe if I get some time I will do a PR to fix, but for now I just wanted to give people the heads up.

**Comment by jimmynguyc** [comment]
@smikkelsen thanks for your feedback !! I'm not maintaining this project anymore since I'm not using it for any projects that I'm working on. Which makes it much harder for me to check and test things. I'll be happy to give you write access if you have plans to contribute to this project :)

**Comment by jimmynguyc** [comment]
> Also, on a side note, and actually the reason I could not use this gem in the first place is the "add_sequences" method doesn't actually add sequences, it replaces them with your new list that you pass it. This in my mind is not desired behavior. It should work just like the add_tags.

@smikkelsen When I made this they haven't got the `object/sequence` endpoint out yet. I've made a branch `add-sequences-to-contacts` and PR (https://github.com/jimmynguyc/ontraport_api/pull/2) with this implementation (blindly, since I have no working API key at the moment). Can you help and take a look?

**Comment by smikkelsen** [comment]
Yeah, I wouldn't be opposed to updating it. It might be a little bit till I can get to it (under the gun to finish up some work, and its consuming all of my free time at the moment).

Also, just as a followup, I got an email back from Ontraport this morning finally admitting that their change shouldn't have happened, and they rolled it back. So the broken endpoint I mentioned before should be back to normal now.

I can look at your PR also when I get a min. in the meantime if you need a working example, I forked another gem and added a sequence method which will add to instead of replace.

https://github.com/smikkelsen/ontraport-ruby/blob/master/lib/ontraport.rb line 197

Thanks!

**Comment by jimmynguyc** [comment]
> It might be a little bit till I can get to it (under the gun to finish up some work, and its consuming all of my free time at the moment).

No worries :) Like I said I'm not actively using this gem anymore so if you're still finding this gem useful go ahead and build on top of it.

> in the meantime if you need a working example, I forked another gem and added a sequence method which will add to instead of replace. https://github.com/smikkelsen/ontraport-ruby/blob/master/lib/ontraport.rb line 197

Yup that's the same endpoint I'm using for the new updates.
